testproject/testproject/urls.py

```diff
@@ -22,6 +22,18 @@
         ),
     ]
 
+if settings.DEBUG:
+    try:
+        import debug_toolbar
+        urlpatterns = [
+            re_path('__debug__/', include(debug_toolbar.urls)),
+
+            # For django versions before 2.0:
+            # url(r'^__debug__/', include(debug_toolbar.urls)),
+
+        ] + urlpatterns
+    except Exception as e:
```

Comment:
We should probably catch the `ImportError` here and not all exception types?

Reply:
Ok.